api/app_factory.py

```
@@ -253,7 +253,7 @@ async def handle_oauth_error(
 
     # Serve React app for all non-API routes (SPA catch-all)
     @app.get("/{full_path:path}", include_in_schema=False)
-    async def serve_react_app(_full_path: str):
+    async def serve_react_app(full_path: str):
```

Ruff (0.14.10)

Unused function argument: `full_path`

(ARG001)